eng/bomgenerator/generateAzureSDKBOM.ps1

```diff
@@ -36,13 +36,14 @@ function UpdateBomProjectElement($OldPomFilePath, $NewPomFilePath) {
 Write-Output "InputDir:$($InputDir)"
 Write-Output "OutputDir:$($OutputDir)"
 Write-Output "Updating version_client.txt file by looking at the packages released to maven."
+# TODO (alzimmer): Handle other group IDs for generating BOMs
```

Comment:
please file a tracking `Azure.Core.V2` issue for this and reference the issue number